docs/core_docs/docs/tutorials/chatbot.ipynb

```diff
@@ -1,1149 +1,1448 @@
-        "const callModel3 = async (state: typeof GraphAnnotation.State) => {\n",
-        "  const chain = prompt2.pipe(llm);\n",
-        "  const response = await chain.invoke(state);\n",
-        "  return { messages: [response] };\n",
-        "};\n",
-        "\n",
-        "const workflow3 = new StateGraph(GraphAnnotation)\n",
-        "  .addNode(\"model\", callModel3)\n",
-        "  .addEdge(START, \"model\")\n",
-        "  .addEdge(\"model\", END);\n",
-        "\n",
-        "const app3 = workflow3.compile({ checkpointer: new MemorySaver() });"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": 43,
-      "metadata": {},
-      "outputs": [
-        {
-          "name": "stdout",
-          "output_type": "stream",
-          "text": [
-            "AIMessage {\n",
-            "  \"id\": \"chatcmpl-ABUXkq2ZV9xmOBSM2iJbYSn8Epvqa\",\n",
-            "  \"content\": \"¡Hola, Bob! ¿En qué puedo ayudarte hoy?\",\n",
-            "  \"additional_kwargs\": {},\n",
-            "  \"response_metadata\": {\n",
-            "    \"tokenUsage\": {\n",
-            "      \"completionTokens\": 12,\n",
-            "      \"promptTokens\": 32,\n",
-            "      \"totalTokens\": 44\n",
-            "    },\n",
-            "    \"finish_reason\": \"stop\",\n",
-            "    \"system_fingerprint\": \"fp_1bb46167f9\"\n",
-            "  },\n",
-            "  \"tool_calls\": [],\n",
-            "  \"invalid_tool_calls\": [],\n",
-            "  \"usage_metadata\": {\n",
-            "    \"input_tokens\": 32,\n",
-            "    \"output_tokens\": 12,\n",
-            "    \"total_tokens\": 44\n",
-            "  }\n",
-            "}\n"
-          ]
-        }
-      ],
-      "source": [
-        "const config4 = { configurable: { thread_id: uuidv4() } }\n",
-        "const input6 = {\n",
-        "  messages: [\n",
-        "    {\n",
-        "      role: \"user\",\n",
-        "      content:  \"Hi im bob\"\n",
-        "    }\n",
-        "  ],\n",
-        "  language: \"Spanish\"\n",
-        "}\n",
-        "const output7 = await app3.invoke(input6, config4)\n",
-        "console.log(output7.messages[output7.messages.length - 1]);"
-      ]
-    },
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "vscode": {
+     "languageId": "raw"
+    }
+   },
+   "source": [
+    "---\n",
+    "sidebar_position: 1\n",
+    "keywords: [conversationchain]\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Build a Chatbot\n",
+    "\n",
+    "\n",
+    ":::info Prerequisites\n",
+    "\n",
+    "This guide assumes familiarity with the following concepts:\n",
+    "\n",
+    "- [Chat Models](/docs/concepts/chat_models)\n",
+    "- [Prompt Templates](/docs/concepts/prompt_templates)\n",
+    "- [Chat History](/docs/concepts/chat_history)\n",
+    "\n",
+    "This guide requires `langgraph >= 0.2.28`.\n",
```

Comment:
If we are moving all this to use LangGraph, should we just work towards combine docs?

Not in this PR obviously but in general